code_review.py

```diff
@@ -0,0 +1,113 @@
+# -*- coding: utf-8 -*-
+import os, sys, traceback, logging, configparser
+import xlsxwriter
+from datetime import datetime, timedelta, timezone
+from apscheduler.schedulers.blocking import BlockingScheduler
+from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from logging.handlers import RotatingFileHandler
+
+
+def main(argv):
+    greetings()
+
+
+
+    print('Press Crtl+{0} to exit'.format('Break' if os.name == 'nt' else 'C'))
+
+
+
+    app = Flask(__name__)
+    handler = RotatingFileHandler('bot.log', maxBytes=10000, backupCount=1)
+    handler.setLevel(logging.INFO)
+    app.logger.addHandler(handler)
+    app.config['SQLALCHEMY_DATABASE_URI'] = 'postgresql+psycopg2://postgres:123mudar@127.0.0.1:5432/bot_db'
+    db = SQLAlchemy(app)
+    config = configparser.ConfigParser()
+    config.read('/tmp/bot/settings/config.ini')
+
+
+
+    var1 = int(config.get('scheduler','IntervalInMinutes'))
+    app.logger.warning('Intervalo entre as execucoes do processo: {}'.format(var1))
+    scheduler = BlockingScheduler()
+
+
+
+    task1_instance = scheduler.add_job(task1(db), 'interval', id='task1_job', minutes=var1)
```

Comment:
Acho que não precisamos dessa instance
```suggestion
    scheduler.add_job(task1(db), 'interval', id='task1_job', minutes=var1)
```